amp/repository/aim/view/editOrganisation.jsp

```diff
@@ -1,2198 +1,2198 @@
-                                                                            <html:optionsCollection property="fiscalCal" label="name" value="ampFiscalCalId"/>
-                                                                        </html:select>
-                                                                    </td>
-                                                                </tr>
-                                                                <tr>
-                                                                    <td  style="text-align:right" class="tdBoldClass">
-                                                                    	<digi:trn>Sectors Scheme</digi:trn>
-                                                                    	<field:display name="Mandatory Indicator For Sector Preferences" feature="NGO Form">
-                                                                    		<span id="mandatorySectScheme"><font color="red">*</font></span>
-                                                                    	</field:display>
-                                                                    </td>
-                                                                    <td>
-                                                                        <html:select property="ampSecSchemeId" styleClass="selectStyle" styleId="ampSecSchemeId">
-                                                                            <c:set var="translation">
-                                                                                <digi:trn>Sectors Scheme</digi:trn>
-                                                                            </c:set>
-                                                                            <html:option value="-1">-- ${translation} --</html:option>
-                                                                            <logic:notEmpty name="aimAddOrgForm" property="sectorScheme">
-                                                                                <html:optionsCollection name="aimAddOrgForm" property="sectorScheme" value="ampSecSchemeId" label="secSchemeName" />
-                                                                            </logic:notEmpty>
-                                                                        </html:select>
-                                                                    </td>
-                                                                </tr>
-                                                                <tr>
-                                                                    <td style=" text-align:right" class="tdBoldClass">
-                                                                    	<digi:trn>Sector Preferences</digi:trn>
-                                                                    	<field:display name="Mandatory Indicator For Sector Preferences" feature="NGO Form">
-                                              
```

Comment:
line 1070 should be renamed to admin level 1 (FM entry) with its translation

Reply:
Line 1078 change Specify Region to specify admin level 1 (and fix its translation)

Reply:
I think the idea of a regional organization is a little bit different from administrative levels.